sync-data-to-another-database/scripts/count_data.js

```diff
@@ -0,0 +1,30 @@
+const
+  Kuzzle = require('kuzzle-sdk'),
+  Cassandra = require('cassandra-driver'),
+  fs = require('fs');
+
+const cassandraClient = new Cassandra.Client({ contactPoints: [ 'cassandra', 'localhost' ] });
+const kuzzle = new Kuzzle('localhost', { port: 7512 })
+
+kuzzle
```

Comment:
Consider flattening your promises calls, for instance:

```
kuzzle
  .collection('yellow-taxi', 'nyc-open-data')
  .countPromise({})
  .then(result => {
    console.log(`Kuzzle: Total documents in nyc-open-data.yellow-taxi : ${result}`)

    return cassandraClient.execute('SELECT COUNT(*) FROM nyc_open_data.yellow_taxi');
  })
  .then(result => {
    console.log(`Cassandra: Total lines in nyc_open_data.yellow_taxi : ${result.rows[].count}`);
    return kuzzle.disconnect();
  })
  .catch(error => {
    console.log(`Error: ${error}`)
    process.exit(1);
  });
```

Moreover, I think that force-exit with a status code of 0 is not clean. I would rather call `kuzzle.disconnect()` to close the network connection, that should be enough to empty the event loop, and at that point Node.js should exit by itself.

(you may have to do the same with the Cassandra client)